src/term/mod.rs

```diff
@@ -2053,6 +2056,80 @@ mod tests {
 
         assert_eq!(term.grid()[&cursor].c, '▒');
     }
+
+    #[test]
+    fn increase_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+        term.change_font_size(10.0);
+
+        let expected_font_size :Size = config.font().size() + Size::new(10.0);
+        assert_eq!(term.font_size, expected_font_size);
+    }
+
+    #[test]
+    fn decrease_font_size_works() {
+        let size = SizeInfo {
+            width: 21.0,
+            height: 51.0,
+            cell_width: 3.0,
+            cell_height: 3.0,
+            padding_x: 0.0,
+            padding_y: 0.0,
+        };
+        let config : Config = Default::default();
+        let mut term = Term::new(&config, size);
+        let negative_font_size_step :f32 = -10.0;
```

Comment:
I think for the sake of unification, the increase and decrease font size method should probably look the same. So you would have to either add `positive_font_size_step` to the `increase_font_size_works` test, or remove it from this method.

Since the increase and decrease methods are so similar, it might be a good idea to just define a method `change_font_size_works(change: f32)` and then call it from both increase and decrease font size tests.

Reply:
Done!